src/RecoveryServices/RecoveryServices.Backup.Autorest/custom/Cmdlets/Policy/Edit-AzrecoveryServicesBackupRetentionPolicyClientObject.ps1

```diff
@@ -0,0 +1,1023 @@
+                       if(($policyObject.SubProtectionPolicy[$Index].RetentionPolicy.RetentionDurationCount -eq $null) -or ($policyObject.SubProtectionPolicy[$Index].RetentionPolicy.RetentionDurationCount -eq 0))
+                       {
+                           $policyObject.SubProtectionPolicy[$Index].RetentionPolicy = [Microsoft.Azure.PowerShell.Cmdlets.RecoveryServices.Models.Api20230201.SimpleRetentionPolicy]::new()
+                           $policyObject.SubProtectionPolicy[$Index].RetentionPolicy.Type = "SimpleRetentionPolicy"
+                           $policyObject.SubProtectionPolicy[$Index].RetentionPolicyType = "SimpleRetentionPolicy"
+                           $policyObject.SubProtectionPolicy[$Index].RetentionPolicy.RetentionDurationType = "Days"
+                           $policyObject.SubProtectionPolicy[$Index].RetentionPolicy.RetentionDurationCount = $DifferentialRetentionPeriodInDays
+                       }
+                       elseif(($policyObject.SubProtectionPolicy[$Index].RetentionPolicy.RetentionDurationCount -ne $null -or $policyObject.SubProtectionPolicy[$Index].RetentionPolicy.RetentionDurationCount -ne 0) -and ($DifferentialRetentionPeriodInDays -ne $null) )
+                       {
+                           $policyObject.SubProtectionPolicy[$Index].RetentionPolicy.RetentionDurationCount = $DifferentialRetentionPeriodInDays
+                       }
+                       $policyObject.SubProtectionPolicy[$Index].RetentionPolicy.RetentionDurationType="Days"
+                   }                   
+               }
+               #Write-Host "Retention period for differential backups: $($policyObject.SubProtectionPolicy[$Index].RetentionPolicy.RetentionDurationCount)"
+          }
+          
+          if($ModifyIncrementalBackup)
+          {
+               # Mandatory Full backup conditions for testing
+               $policyObject.SubProtectionPolicy[0].SchedulePolicy.ScheduleRunFrequency="Weekly"
+               $policyObject.SubProtectionPolicy[0].SchedulePolicy.ScheduleRunDay=@("Monday","Tuesday")
+               $policyObject.SubProtectionPolicy[0].SchedulePolicy.scheduleRunTime=@("2020-09-30T19:30:00Z")
+               $policyObject.SubProtectionPolicy[0].RetentionPolicy.MonthlySchedule.RetentionScheduleWeekly.DaysOfTheWeek=@("Monday","Tuesday")
+               $policyObject.SubProtectionPolicy[0].RetentionPolicy.YearlySchedule.RetentionScheduleWeekly.DaysOfTheWeek=@("Monday","Tuesday")
+               $policyObject.SubProtectionPolicy[0].RetentionPolicy.WeeklySchedule.DaysOfTheWeek=@("Monday","Tuesday")
+               # Schedule Policy values for testing
+               $policyObject.SubProtectionPolicy += [Microsoft.Azure.PowerShell.Cmdlets.RecoveryServices.Models.Api20230201.SubProtectionPolicy]::new()
+               $DifferentialPolicy = $policyObject.SubProtectionPolicy[$policyObject.SubProtectionPolicy.Length - 1]
```

Comment:
Pls use this approach to use differential, inremental policy
 $DifferentialPolicy = $policyObject.SubProtectionPolicy | where { $_.PolicyType -match "Differential" }
 $IncrementalPolicy = $policyObject.SubProtectionPolicy | where { $_.PolicyType -match "Incremental" }